assets/css/style.css

```diff
@@ -1701,6 +1701,7 @@ margin: -20px 0px 50px 0px;
 }
 .search-form-item{
   padding-bottom: 20px;
+  font-family: ;
```

Comment:
Only thing I see is the missing font-family

Reply:
The new font weight is in the HTML from the google link

Reply:
Sorry this must have been an error- just deleted it!